airflow/secrets/base_secrets.py

```diff
@@ -73,3 +75,12 @@ def get_variable(self, key: str) -> Optional[str]:
         :return: Variable Value
         """
         raise NotImplementedError()
+
+    def get_config(self, key: str) -> Optional[str]:    # pylint: disable=unused-argument
```

Comment:
I think this should be

```suggestion
    def get_config(self, section: str, key: str) -> Optional[str]:    # pylint: disable=unused-argument
```

Reply:
Oh, except it comes from a single value in the config file. Hmmmm.

Perhaps update the example to show include a `/` in it? 

```ini
[core]
sql_alchemy_conn_secret = core/sql_alchemy_conn
```

Dunno.

Reply:
add docs in https://github.com/apache/airflow/pull/9645/commits/032d33f257681ac9d189357fe90fdba23a583da7